src/Robo/Commands/Acsf/AcsfCommand.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Acquia\Blt\Robo\Commands\Acsf;
+
+use Acquia\Blt\Robo\BltTasks;
+use Robo\Contract\VerbosityThresholdInterface;
+
+/**
+ * Defines commands in the "acsf" namespace.
+ */
+class AcsfCommand extends BltTasks {
+
+  /**
+   * Initializes ACSF support for project.
+   *
+   * @command acsf:init
+   *
+   * @options acsf-package
+   */
+  public function acsfInitialize($options = ['acsf-package' => 'drupal/acsf:^1.33.0']) {
+    $this->acsfHooksInitialize();
+
+    $this->say('Adding acsf module as a dependency.');
+    $this->requireAcsf($options['acsf-package']);
+
+    $this->say("In the future, you may pass a custom value for acsf.package to override default version. E.g., blt acsf:init --acsf.package='drupal/acsf:8.1.x-dev'");
+
+    $this->acsfDrushInitialize();
+  }
+
+  /**
+   * Refreshes the ACSF settings and hook files.
+   *
+   * @command acsf:init:drush
+   */
+  public function acsfDrushInitialize() {
+    $drushBin = $this->getConfigValue('drush.bin');
+    $drushAlias = $this->getConfigValue('drush.default_alias');
```

Comment:
Hmm, is there actually a `drush.alias` config option or are you saying we'd be defining a new key value pair in config essentially? I didn't find one in our config files ([`build.yml`](https://github.com/acquia/blt/blob/8.x/phing/build.yml#L84), [`project.yml`](https://github.com/acquia/blt/blob/8.x/template/blt/project.yml#L23), `project.local.yml`). 

Also, I don't know if we can access drush alias info in `DefaultConfig` without some refactoring, since [it's instantiated before](https://github.com/acquia/blt/blob/8.x/bin/blt-robo-run.php#L43) `project.yml` and `project.local.yml` are imported. We could add a setter method to `DefaultConfig` and then invoke it after [this line](https://github.com/acquia/blt/blob/8.x/bin/blt-robo-run.php#L49), but that feels hacky.